tasktray.py

```diff
@@ -0,0 +1,61 @@
+from PyQt5.QtWidgets import QSystemTrayIcon, QMenu, QAction
+from PyQt5 import QtGui
+from PyQt5.QtCore import Qt
+import settings
+
+
+class TrayWidget(QSystemTrayIcon):
+
+    def __init__(self, parent=None):
+        super().__init__(parent)
+        self.parent = parent
+        self.create_actions()
+        self.create_menu()
+        self.create_icon()
+
+    def create_actions(self):
+
+        # Acton - Show main widget (menu and double click the tray icon)
+        self.showMainWidgetAction = QAction(self)
+        self.showMainWidgetAction.setText(self.tr("Show &Main Widget"))
```

Comment:
Does this text work as expected without updating translation files?

Reply:
Oh, maybe `tr` is the method from `QSystemTrayIcon`

Reply:
These were remnants of a copy and paste of the sample code...
I would like to use the original `lang/locale.json` instead of `QSystemTrayIcon.tr()` ,
because defining resources and IDs using `tr()` seems very [cumbersome](https://doc.qt.io/archives/qt-4.8/i18n-source-translation.html).
So I'm removing `tr()` functions.

(I'm going to implement the locale switching feature in the other pull request)